### PR TITLE
Us4 event details page

### PR DESCRIPTION
**Pull request by n377i**

### CODE REVIEW

File: components/CategoryTag/CategoryTag.styled.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Button = styled.button`
```
Comment (MerleMak) [inline components/CategoryTag/CategoryTag.styled.js:3]: I don't think this needs to be a button (yet). 
As far as I can see there is no user interaction planned for the tag at this point. So its not necessary to use a clickable element, a span or a div would work the same and don't specify the utility:)
```suggestion
export const Button = styled.span`
```

File: pages/events/[id]/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { useRouter } from "next/router";
+import { events } from "@/lib/events";
```
Comment (MerleMak) [inline pages/events/[id]/index.js:2]: if I am not mistaken, Bruno's US with the database connection (including the API route for the events) is already merged to main.

Maybe it makes sense for you to merge main into your branch and already receive the events from the DB, so you don't have to open another branch for that.

But I am sure you guys got it figured out, I just wanted to make sure its not forgotten :)

**Review by MerleMak — APPROVED** [review]
Very very nice Annette!
I only left you two small comments, you might want to consider. 
Great job, keep it up!

File: pages/events/[id]/index.js (hunk 2 above)
Comment (Bruno0221) [reply pages/events/[id]/index.js:2]: the API route is a separate User Story. Thanks for the reminder though :)

File: components/CategoryTag/CategoryTag.styled.js (hunk 1 above)
Comment (MerleMak) [reply components/CategoryTag/CategoryTag.styled.js:3]: any thoughts?

Comment (n377i) [reply components/CategoryTag/CategoryTag.styled.js]: I changed it to span. Thanks.

**Pull request merged**